[2024-11-06T16:34:12.154+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-11-06T16:34:12.229+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_postgess_v2.delete_in_postgres_table manual__2024-11-06T16:26:25.203907+00:00 [queued]>
[2024-11-06T16:34:12.252+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_postgess_v2.delete_in_postgres_table manual__2024-11-06T16:26:25.203907+00:00 [queued]>
[2024-11-06T16:34:12.253+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 3
[2024-11-06T16:34:15.635+0000] {taskinstance.py:2889} INFO - Executing <Task(PostgresOperator): delete_in_postgres_table> on 2024-11-06 16:26:25.203907+00:00
[2024-11-06T16:34:15.728+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=12410) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-11-06T16:34:15.741+0000] {standard_task_runner.py:72} INFO - Started process 12421 to run task
[2024-11-06T16:34:15.736+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgess_v2', 'delete_in_postgres_table', 'manual__2024-11-06T16:26:25.203907+00:00', '--job-id', '59', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_postgress.py', '--cfg-path', '/tmp/tmpqivba68a']
[2024-11-06T16:34:15.790+0000] {standard_task_runner.py:105} INFO - Job 59: Subtask delete_in_postgres_table
[2024-11-06T16:34:15.898+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:209 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-11-06T16:34:16.201+0000] {task_command.py:467} INFO - Running <TaskInstance: dag_with_postgess_v2.delete_in_postgres_table manual__2024-11-06T16:26:25.203907+00:00 [running]> on host 61aa06b65713
[2024-11-06T16:34:18.903+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Adithya' AIRFLOW_CTX_DAG_ID='dag_with_postgess_v2' AIRFLOW_CTX_TASK_ID='delete_in_postgres_table' AIRFLOW_CTX_EXECUTION_DATE='2024-11-06T16:26:25.203907+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-11-06T16:26:25.203907+00:00'
[2024-11-06T16:34:18.906+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-11-06T16:34:18.948+0000] {sql.py:278} INFO - Executing: 
delete from dag_runs where  dt='2024-11-06' and dag_id = 'dag_with_postgess_v2'
[2024-11-06T16:34:18.969+0000] {base.py:84} INFO - Retrieving connection 'postgres_localhost'
[2024-11-06T16:34:18.990+0000] {base.py:84} INFO - Retrieving connection 'postgres_localhost'
[2024-11-06T16:34:19.053+0000] {sql.py:544} INFO - Running statement: 
delete from dag_runs where  dt='2024-11-06' and dag_id = 'dag_with_postgess_v2', parameters: None
[2024-11-06T16:34:19.075+0000] {sql.py:553} INFO - Rows affected: 0
[2024-11-06T16:34:19.123+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-11-06T16:34:19.125+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=dag_with_postgess_v2, task_id=delete_in_postgres_table, run_id=manual__2024-11-06T16:26:25.203907+00:00, execution_date=20241106T162625, start_date=20241106T163412, end_date=20241106T163419
[2024-11-06T16:34:19.358+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-11-06T16:34:19.424+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-11-06T16:34:19.429+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
